src/services/apimap-field-builder.js

```diff
@@ -143,9 +143,8 @@ function ApimapFieldBuilder(model, column, options) {
       columnName: column.field,
     };
 
-    if (column.primaryKey === true) {
-      schema.primaryKey = true;
-    }
+    schema.isPrimaryKey = Boolean(column.primaryKey);
+
     if (schema.type === 'Enum') {
       schema.enums = column.values;
     }
```

Comment:
`enums` / `isRequired` properties are not always set but we can retrieve them in each field definition in the `.forestadmin-schema.json`.